packages/roosterjs-plugin-picker/lib/PickerPlugin.ts

```diff
@@ -408,12 +434,60 @@ export default class PickerPlugin<T extends PickerDataProvider = PickerDataProvi
                 if (nodeId && nodeId.indexOf(this.pickerOptions.elementIdPrefix) == 0) {
                     let replacementNode = this.dataProvider.onRemove(nodeAfterCursor, false);
                     this.replaceNode(nodeAfterCursor, replacementNode);
-                    this.handleKeyDownEvent(event);
+                    this.cancelDefaultKeyDownEvent(event);
                 }
             }
         }
     }
 
+    private onAndroidInputEvent(event: PluginInputEvent) {
+        this.newInputLength = this.calcInputLength(event);
+
+        if (
+            this.newInputLength < this.currentInputLength ||

```

Comment:
If cursor is at beginning of a node, and backspace will actually move cursor to the previous node. In that case current node length will be greater than originl length (0). Can the code handle that case? This is a question to John.

Reply:
No sure I understand. How can I test this behavior you're describing?

Reply:
[NODE 1][[CURSOR] NODE 2]

When cursor is in front of node 2, press Backspace will still delete the last char of node 1. But in that case node length we measure before and after the event are for different node (firstly node 2, then node 1). In that case comparing the length doesn't seem make sense.

But it might be a corner case. I'll leave this to John, you can decide if this is an real issue or it can never happen.

Reply:
I test the scenario where you have two mentions next to each other with no space and you put the cursor between them and hit backspace. Everything works.